README.md

```diff
@@ -235,10 +235,17 @@ commit*), and then build the backend as described
 TRITON_TENSORFLOW_DOCKER_IMAGE set to refer to the new Docker image.
 
 
-## Using the Tensorflow Backend
+## Using the TensorFlow Backend
+### Platform
+
+TensorFlow recognizes two kinds of model format, GraphDef and SavedModel. To
+differentiate the model format, 'platform' in model's 'config.pbtxt' file will
+be used, "tensorflow_graphdef" should be set for GraphDef format and
+"tensorflow_savedmodel" should be set for SavedModel format.
+
```

Comment:
Can you change to something like this?

---


TensorFlow recognizes two kinds of model format, GraphDef and SavedModel. To
differentiate the model format, specify the appropriate `platform` in the model's
`config.pbtxt` such as:
```
# config.pbtxt
platform: "tensorflow_graphdef"
```
or
```
# config.pbtxt
platform: "tensorflow_savedmodel"
```